Specs/Scene/BillboardCollectionSpec.js

```diff
@@ -682,14 +684,16 @@ defineSuite([
     it('adds and renders a billboard', function() {
         billboards.add({
             position : Cartesian3.ZERO,
-            image : greenImage
+            image : greenImage,
+            disableDepthTestDistance : 0.0
```

Comment:
Why do you have to explicitly set `disableDepthTestDistance` for this to pass?

Reply:
I wouldn't think we have to, but when they're both set to the default `5000` the wrong one renders in front.  Should I open an issue?  I'll see if I can reproduce it for a normal case.  Would it be weird for this since there's no globe and it's at `0,0,0`?

Reply:
@bagnell yeah it looks like it's a bug with `disableDepthTestDistance`.  I can reproduce it pretty easily, I'll open a new issue

Reply:
I think it's because the `disableDepthTestDistance` is greater than the distance from the camera to the billboard